tests/dummy/config/content-security-policy.js

```diff
@@ -0,0 +1,52 @@
+module.exports = function() {
+  return {
+    policy: {
+      'default-src': ['\'none\''],
+      'script-src': [
+        '\'self\'',
+        // script tag injected by ember-cli-addon-docs
+        '\'sha256-vf+ZuKOhXq6af1D+7eWhmYHkEHMV8Z75nTtHTR7oFcg=\'',
+        // Instanbul used by ember-cli-code-coverage uses eval by default through
+        // `new Function('return this')`. It provides a configuratoin option
+        // to prevent this but that one is not exposed by ember-cli-code-coverage
+        // yet.
+        // Additionally it injects a script tag into the page.
+        // Please see details in this GitHub issue:
+        // https://github.com/kategengler/ember-cli-code-coverage/issues/214#issuecomment-619398136
+        process.env.COVERAGE ? '\'unsafe-eval\'' : null,
+        process.env.COVERAGE ? '\'sha256-962C3YD3u+SHbZpl2H9WlCU6FY6H8TZ7smnnVmRMtic=\'' : null
```

Comment:
I needed to change this hash after rebase. I guess the rebased caused a newer version of Istanbul to be used. It seems as if that the content of the script tag injected has changed between the two Istanbul versions. Not sure how stable it will be overtime. Maybe adding `'unsafe-inline'` instead would be more stable long term.